README.md

```diff
@@ -752,38 +769,67 @@ Apart from these new fields, the session history entries which correspond to `Ap
 
 _TODO: actually, we should probably expose scroll restoration mode, like `history.scrollRestoration`? That API has legitimate use cases, and we'd like to allow people to never touch `window.history`..._
 
-Finally, all the higher-level mechanisms of session history entry management, such as the interaction with navigation, continue to work as they did before; the correspondence to `AppHistoryEntry` APIs does not change the processing there.
+### Correspondence with the joint session history
 
-To understand how navigation interception and queuing interacts with the existing navigation spec, see [the navigation types appendix](#appendix-types-of-navigations). Also note the open questions in [#19](https://github.com/WICG/app-history/issues/19).
-
-## Impact on the back button and user agent UI
+The view of history which the user sees, and which is traversable with existing APIs like `history.go()`, is the joint session history.
 
-The app history API doesn't change anything about how user agents implement their UI: it's really about developer-facing affordances. Users still care about the joint session history, and so that will continue to be presented in UI surfaces like holding down the back button. Similarly, pressing the back button will continue to navigate through the joint session history, potentially across origins and out of the current app history (into a new app history, on the new origin).
+Unlike the view of history presented by `window.history`, `window.appHistory` only gives a view onto session history entries for the current browsing context. This view does not present the joint session history, i.e. it is not impacted by frames. Notably, this means `appHistory.entries.length` is likely to be quite different from `history.length`.
```

Comment:
It seems fine to expose entries that cross browsing contexts via COOP+COEP

Reply:
Yeah, I'll tweak this to "browsing session" I guess? It's not precise.